### ISSUE TITLE
Usage of "clicks" instead of "selects" in the example

### ISSUE THREAD

**Issue by chanakavp** [MicrosoftDocs/microsoft-style-guide#404]
> "When the user **clicks** OK, the transaction is committed. (in content for developers)"

In the above example, the verb **clicks** should be **selects**, as per advice given in https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e98fcd08-8edf-0a1d-2a62-6d827bc327b9
* Version Independent ID: d1ca993e-79e8-5373-44f4-c7390a736bfe
* Content: [Verbs - Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/grammar/verbs)
* Content Source: [styleguide/grammar/verbs.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/main/styleguide/grammar/verbs.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by LauraDSchneider** [comment]
I agree with @chanakavp ! Also, search the whole guide for "click" and replace with "select"; there are probably many instances of this. Thank you!
